gopherpolicy/pkg.go

```diff
@@ -242,6 +244,7 @@ func (t *keystoneToken) ToContext() policy.Context {
 			"tenant_domain_name":          t.ProjectScope.Domain.Name,
 			"application_credential_id":   t.ApplicationCredential.ID,
 			"application_credential_name": t.ApplicationCredential.Name,
+			"is_admin_project":            strconv.FormatBool(t.IsAdminProject),
 			// NOTE: When adding new elements, also adjust the serialization
 			// functions in `serialize.go` as necessary.
```

Comment:
This comment should be heeded, and a testcase should be added demonstrating serialization of token responses that set this flag.